helloworld-rs/src/main/java/org/jboss/as/quickstarts/helloworldrs/JakartaRESTActivator.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.jboss.as.quickstarts.rshelloworld;
+
+import jakarta.ws.rs.ApplicationPath;
+import jakarta.ws.rs.core.Application;
+
+/**
+ * JakartaRESTActivator is an arbitrary name, what is important is that jakarta.ws.rs.core.Application is extended
+ * and the @ApplicationPath annotation is used with a "rest" path.  Without this the rest routes linked to
```

Comment:
Minor, but instead of "rest routes" I think we should use "rest endpoints".

Reply:
done